src/rt/util/utf.d

```diff
@@ -566,7 +565,7 @@ Checks to see if string is well formed or not. $(D S) can be an array
  of $(D char), $(D wchar), or $(D dchar). Throws a $(D UtfException)
  if it is not. Use to check all untrusted input for correctness.
  */
-void validate(S)(in S s)
+void validate(S)(in S s) pure nothrow @safe @nogc
```

Comment:
You're right, it's a different validate, but this one shouldn't be nothrow and we still need UTFExcwption for validation.
BTW the signature of validate should become `inout(char)[] validate(input(ubyte)[] str);`.